.github/actions/setup-tauri-v2/action.yml

```diff
@@ -5,29 +5,55 @@ inputs:
   runtime:
     description: "Whether to also set up runtime dependencies"
     required: false
-    default: false
+    default: "false"
```

Comment:
Just want to make sure this was intentionally being changed to a string from a boolean.

Reply:
Correct - this was an editor lint that clued me into the fix. The `default` key only accepts strings.

Reply:
👍